client/tools/mgr-push/Makefile.python

```diff
@@ -2,18 +2,46 @@ THIS_MAKEFILE := $(realpath $(lastword $(MAKEFILE_LIST)))
 CURRENT_DIR := $(dir $(THIS_MAKEFILE))
 include $(CURRENT_DIR)../../../rel-eng/Makefile.python
 
+MGRPUSH_ROOT_DIR = /manager/client/tools/mgr-push
+RHNPUSH_PACKAGE_DIR = $(MGRPUSH_ROOT_DIR)/rhnpush
+
 # Docker tests variables
+DOCKER_REF            = $(DOCKER_REGISTRY)/$(DOCKER_IMAGE)
 DOCKER_CONTAINER_BASE = systemsmanagement/uyuni/master/docker/containers/uyuni-master
 DOCKER_REGISTRY       = registry.opensuse.org
-DOCKER_RUN_EXPORT     = "PYTHONPATH=$PYTHONPATH"
+DOCKER_RUN_EXPORT     = "PYTHONPATH=/manager/client/tools/mgr-push/rhnpush:/manager/python/:/manager/client/tools/mgr-push:/manager/client/rhel/spacewalk-client-tools/src/"
 DOCKER_VOLUMES        = -v "$(CURDIR)/../../../:/manager"
 
+define setup_structure
```

Comment:
I feel like this is a step in the wrong direction. We have a problem with our Python packages: we can't install them into a virtual environment to run tests against (like any well maintained Python project does), and while this workaround let's us run the tests, it does not fix the underlying issue.

Have you considered fixing this in-tree with a [`src` layout](https://hynek.me/articles/testing-packaging/)? That's what many projects use successfully.

Reply:
Would you consider reasonable to create a follow-up card about this issue? I believe this would take a longer time to be properly solved.

Reply:
Yes, it's reasonable to change the way we run the tests in a follow-up pull request. In the meantime, please make sure we run the new unit tests in Jenkins or via GitHub Actions.